packages/unix-errno/unix-errno.0.6.2/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Unix errno types, maps, and support"
+description: """\
+unix-errno can be used with or without ctypes and OCaml's Unix
+module. Without ctypes and Unix, the basic types and functions are
+provided as well as Errno_host containing errno maps for popular
+operating systems. The errno-srcgen tool for generating OCaml source
+representing Errno.Host.t values will also be built. With ctypes and
+Unix, you'll also receive the errno-map tool for outputting the current
+host's errno map and the Errno_unix module containing an errno global
+variable checking function and Unix.error type converters."""
+maintainer: "Xapi Project"
+authors: "David Sheets <sheets@alum.mit.edu>"
+license: "ISC"
+tags: ["errno" "errno.h" "errors" "unix" "syscall"]
+homepage: "https://github.com/xapi-project/ocaml-unix-errno"
+bug-reports: "https://github.com/xapi-project/ocaml-unix-errno/issues"
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "dune" {>= "2.8"}
+  "alcotest" {with-test}
+  "result"
+  "ctypes"
+]
+depopts: ["base-unix"]
+conflicts: [
+  "ctypes" {< "0.7.0"}
+]
```

Comment:
Simplifies to:
```suggestion
  "ctypes" {>= "0.7.0"}
]
```
also `unix` is always required per https://github.com/xapi-project/ocaml-unix-errno/blob/c91d9ac97e33f7df0d63f95110130130d040c6ce/unix/dune